src/components/migrationToV3.stories.mdx

```diff
@@ -560,15 +699,15 @@ const MyChild = () => {
 
 ### Device Selection APIs Throw Error When Failed
 
-In V2, we added `selectDeviceError` to `useAudioInputs` and `useVideoInputs` hook to get the error of `selectAudioInputDevice` and `selectVideoInputDevice` API from client side when they failed. In V3, we remove `selectDeviceError` from these hooks and throw error directly when `selectAudioInputDevice`, `selectVideoInputDevice`, and `selectAudioOutputDevice` methods of `MeetingManager` failed. The device selection functions return by `useSelectAudioInputDevice`, `useSelectVideoInputDevice`, and `useSelectAudioOutputDevice` hook now throw errors as well.
+In V2, we added `selectDeviceError` to `useAudioInputs` and `useVideoInputs` hook to get the error of `selectAudioInputDevice` and `selectVideoInputDevice` API from client side when they failed. In V3, we remove `selectDeviceError` from these hooks and throw error directly when `selectAudioInputDevice`, `selectVideoInputDevice`, and `selectAudioOutputDevice` methods of `MeetingManager` failed.
```

Comment:
nit: In V3, we have removed....from `MeetingManager` fail.

Reply:
We also have to make clear in docs the selection just updates the state for the selected device and does not start any audio-video based on selection. One has to call start APIs to start the audio video.

Reply:
It should be `start*Device` API in V3. Added some sentence to clarify the renaming thing.